client/widgets/org-top-stats/stats.template.html.slim

```diff
@@ -0,0 +1,65 @@
+.org-top-stats.panel.panel-default
+  .panel-heading
+    help-button
+      p Help will be there
+    h3.panel-title Top {{ params.top }} of organization DB stats
+  .panel-body
+    .row ng-if='loading && stats.length === 0'
+      .col-md-12
+        p Loading...
+    .row ng-if='!loading && stats.length === 0'
+      .col-md-12
+        p.empty No results match the current filters.
+    .row.row-labels ng-if='stats.length > 0'
+      .col-md-12
+        table.table
+          thead.hidden-md.hidden-sm.hidden-xs
+            tr
+              th.name.db-stats-right-border rowspan=2 Name
+              th.projects.text-center.db-stats-right-border rowspan=2 Projects
+              th.tests.text-center.db-stats-right-border rowspan=2 Tests
+              th.payloads.text-center.db-stats-right-border rowspan=2 Payloads
+              th.results.text-center colspan=3 Results
+            tr
+              th.results-count.text-right Rows
+              th.results-prop.text-right %
+              th.results-trend.text-center Trend
+          thead.hidden-lg
+            tr
+              th.name.db-stats-right-border rowspan=2 Name
+              th.tests.text-center.db-stats-right-border rowspan=2 Tests
+              th.payloads.text-center.db-stats-right-border rowspan=2 Payloads
+              th.results.text-center colspan=2 Results
+            tr
+              th.results-count.text-right Rows
+              th.results-trend.text-center Trend
+          tbody
+            tr ng-repeat='stat in stats'
+              td.name.db-stats-right-border {{ stat | orgName }}
+              td.projects.text-right.db-stats-right-border.hidden-md.hidden-sm.hidden-xs {{ stat.projectsCount | number }}
+              td.tests.text-right.db-stats-right-border {{ stat.testsCount | number }}
+              td.payloads.text-right.db-stats-right-border {{ stat.payloadsCount | number }}
+              td.results-count.text-right {{ stat.resultsCount | number }}
+              td.results-prop.text-right.hidden-md.hidden-sm.hidden-xs {{ stat.resultsProp | number:2 }}%
+              td.results-trend.text-center
+                sparkline ng-if='stat.resultsTrend' data='stat.resultsTrend'
+          tfoot
+            tr
+              td.name.db-stats-right-border Top total
```

Comment:
The meaning would be clearer with "Top 5 total". Also, I would expect this `td` and the next `tr`'s first `td` to be `th` elements instead. It will make it visually clearer that they are totals.